Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -720,45 +698,42 @@ public void ToggleFlowLauncher()
                 Hide();
             }
         }
-
         public void Show()
         {
-            if (_settings.UseSound)
-            {
-                MediaPlayer media = new MediaPlayer();
-                media.Open(new Uri(AppDomain.CurrentDomain.BaseDirectory + "Resources\\open.wav"));
-                media.Play();
-            }
+            ((MainWindow)Application.Current.MainWindow).WindowAnimator();
             MainWindowVisibility = Visibility.Visible;
             WinToggleStatus = true;
-            ((MainWindow)Application.Current.MainWindow).WindowAnimator();
             MainWindowOpacity = 1;
         }
 
         public async void Hide()
         {
-            MainWindowOpacity = 0;
             switch (_settings.LastQueryMode)
             {
                 case LastQueryMode.Empty:
                     ChangeQueryText(string.Empty);
                     MainWindowOpacity = 0; // Trick for no delay
-                    await Task.Delay(100); //Time for change to opacity
+                    await Task.Delay(50); //Time for change to opacity
```

Comment:
As you know, in this case, there should be a delay regardless of animation because the query should be erased.